StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/PropertySummaryDocumentationAnalyzer.cs

```diff
@@ -130,7 +130,10 @@ private static void AnalyzeSummaryElement(SyntaxNodeAnalysisContext context, Xml
             var textElement = summaryElement.Content.FirstOrDefault() as XmlTextSyntax;
             var text = textElement == null ? string.Empty : XmlCommentHelper.GetText(textElement, true).TrimStart();
 
-            bool startsWithGetOrSet = text.StartsWith(startingTextGetsOrSets, StringComparison.OrdinalIgnoreCase);
+            bool prefixIsGetsOrSets = text.StartsWith(startingTextGetsOrSets, StringComparison.OrdinalIgnoreCase);
+            bool prefixIsGets = text.StartsWith(startingTextGets, StringComparison.OrdinalIgnoreCase) && !prefixIsGetsOrSets;
```

Comment:
💭 Reversing the condition (put `!prefixIsGetsOrSets` first) will have a small performance benefit.